Examples/file-manager.lisp

```diff
@@ -0,0 +1,349 @@
+;;;; (C) Copyright 2019, 2020 Jan Moringen
+;;;;
+;;;; This library is free software; you can redistribute it and/or
+;;;; modify it under the terms of the GNU Library General Public
+;;;; License as published by the Free Software Foundation; either
+;;;; version 2 of the License, or (at your option) any later version.
+;;;;
+;;;; This library is distributed in the hope that it will be useful,
+;;;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;;;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+;;;; Library General Public License for more details.
+;;;;
+;;;; You should have received a copy of the GNU Library General Public
+;;;; License along with this library; if not, write to the
+;;;; Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+;;;; Boston, MA  02111-1307  USA.
+
+(defpackage #:mcclim.examples.file-manager
```

Comment:
while I agree that mcclim.examples.file-manager is a compelling package name, I think that we should standardize the package names in our examples to something derived from a clim-demo as something what is already there for a long time. I see that we have some defpackage's in Examples, so these should be changed too.

if you have time for that, please rename packages in Examples to clim-demo.demo-name, if you don't, please only rename the package for file-manager demo to `clim-demo.file-manager` and I'll make a PR which changes package names in other demos too later.

Reply:
Changed.